samples/todo-app/database-api/databaseApi.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
```

Comment:
I tested this with VS 2019 and VS 2022 B2k extensions , both work fine. Only catch with VS 2019 is if they are using this application to test they have to select ```use preview of .NET SDK's. (requires restart)``` in Options in VS2019 would make it work. 

![image](https://user-images.githubusercontent.com/105889062/224138576-b6a9e2eb-76cd-49b5-80d7-f8930c46ab8d.png)

Reply:
Seems preview version fixes the bug we had, that we ran into in December :) very nice!